resources/views/contact.blade.php

```diff
@@ -116,9 +121,13 @@ class="block w-full px-5 py-3 text-center font-medium text-pine-600 bg-gray-50 h
 
 <section id="contact" class="text-center">
     @if (session('success'))
-        <div class="alert alert-success" role="alert">
-            {{ __("Thank you for contacting us! We'll respond as soon as we can.") }}
-        </div>
+    <div class="p-2 bg-pine-300 items-center text-white leading-none lg:rounded-full flex lg:inline-flex" role="alert">
+        <span class="flex rounded-full bg-pine-500 uppercase px-2 py-1 text-xs font-bold mr-3">Sent</span>
+        <span class="font-semibold mr-2 text-left flex-auto">Thank you for contacting us! We'll respond as soon as we can</span>
+        <button id="closeAlertButton" class="fill-current opacity-75 h-4 w-4 outline-none focus:outline-none">
+            <span>×</span>
+        </button>
+    </div>
```

Comment:
These are the changes too make make closing this alert fully functional using only AlpineJS, with a fade-out when closing it as a bonus feature.
```suggestion
    <div x-data="{open: true}" x-show="open" x-transition:leave="transition ease-in duration-300" x-transition:leave-start="opacity-100" x-transition:leave-end="opacity-0" class="p-2 bg-pine-300 items-center text-white leading-none lg:rounded-full flex lg:inline-flex" role="alert">
        <span class="flex rounded-full bg-pine-500 uppercase px-2 py-1 text-xs font-bold mr-3">Sent</span>
        <span class="font-semibold mr-2 text-left flex-auto">Thank you for contacting us! We'll respond as soon as we can</span>
        <button @click="open = false" class="fill-current opacity-75 h-4 w-4 outline-none focus:outline-none">
            <span>×</span>
        </button>
    </div>
```